department-of-veterans-affairs/vets-api
# API-41894 Fix schema to ref BRD for separationLocationCode

## Summary

- *This work is behind a feature toggle (flipper): NO*
- Updates v1 526 schema to reference BRD as the source of location codes for submissions.
- Adds generated swagger for v1.

## Related issue(s)

[API-41894](https://jira.devops.va.gov/browse/API-41894)

## Testing done

- Submit an integer separationLocationCode for a service period with activeDutyEndDate in future and confirm the resulting 422 detail message includes: "Code must match the values returned by the /intake-sites endpoint on the [Benefits reference Data API](https://developer.va.gov/explore/benefits/docs/benefits_reference_data?version=current)."
- View swagger doc locally to confirm the v1 /forms/526 schema includes the above message for separationLocationCode. 

## Screenshots
![Screenshot 2024-11-12 at 10 56 42 AM](https://github.com/user-attachments/assets/aa5b1119-8fe9-4ccb-8a34-3f1ac296f389)
![Screenshot 2024-11-12 at 10 56 23 AM](https://github.com/user-attachments/assets/f902b707-5a0a-45e3-9362-201fa02fbce3)


## What areas of the site does it impact?
v1 526.json schema
v1 swagger documentation

## Acceptance criteria
